.github/PULL_REQUEST_REVIEW_TEMPLATE.md

```diff
@@ -0,0 +1,4 @@
+## Checklist
+While reviewing a Pull Request, the following items must have been completed for the PR to be approved:
+- [ ] Requester has added doc strings to any user-facing functions
+- [ ] Requester has updated the CHANGELOG.md file with your changes
```

Comment:
I would add a item for fulfilling the acceptance criteria of whatever the task is that this PR was meant to accomplish. I realize that this will evolve over time, especially if/when we implement any form of automated testing.

Reply:
Fixed!